.github/ISSUE_TEMPLATE/bug_report_linux.yml

```
@@ -0,0 +1,73 @@
+name: Bug Report (Linux)
+description: File a bug report for Linux.
+labels: ["bug", "triage"]
+projects: ["freedomofpress/dangerzone"]
+body:
+  - type: markdown
+    attributes:
+      value: |
+        Hi, and thanks for taking the time to open this bug report. 
+  - type: textarea
+    id: what-happened
+    attributes:
+      label: What happened?
+      description: Can you specify the steps you followed, so that we can reproduce?
+      value: "A bug happened!"
+    validations:
+      required: true
+  - type: textarea
+    id: os-version
+    attributes:
+      label: operating system version
+      description: |
+        Which version of your operating sytem do you use?
+
+        On Linux, you should have good results with this command `cat /etc/os-release`
```

Since we are already in the Linux bug report, maybe we can be a bit more specific:

```suggestion
      label: Linux distribution
      description: |
        What is the name and version of your Linux distribution? You can find it out with `cat /etc/os-release`
      placeholder: Ubuntu 20.04.6 LTS
```